docs/stark-ui/GETTING_STARTED.md

```diff
@@ -0,0 +1,10 @@
+# Getting started
+
+The recommended way to get started with **Stark** is to download the [starter](../../starter) and run `npm install`.
+
+## Modules
+
+The starter comes with most things preconfigured and all styling needed for stark-ui is imported by default.
+
+In `src/styles/styles.scss` all the styles and theming of the components is imported via `"~@nationalbankbelgium/stark-ui/assets/stark-ui-bundle";`.
+If you want to change the styling of a component you will need to copy all the imports from `stark-ui-bundle.scss` and replace the ones you want to change with your own implementation.
```

Comment:
In this use case where the developer selectively imports the styles, we need to explain which styles are not related to components (I.e. base-theme, base, etc...) that should be imported first

Reply:
# Getting started

The recommended way to get started with **Stark** is to download the [starter](../../starter) and run `npm install`.

## Modules

The starter comes with most things preconfigured and all Sass files needed for stark-ui are imported by default via `~@nationalbankbelgium/stark-ui/assets/stark-ui-bundle`.

You can also pick and choose which parts of the stark styling/theming you want to use. Simply remove `@import "~@nationalbankbelgium/stark-ui/assets/stark-ui-bundle";` from `starter/src/styles/styles.scss` and import only the desired files.
It is advised to at least import `base` and `base-theme` since these provide you with a clean basis and some basics for styling/theming your app.

Original:
```scss
/*
IMPORTANT: Stark styles are provided as SCSS styles so they should be imported in a SCSS file!
*/
@import "theme";
@import "~@nationalbankbelgium/stark-ui/assets/stark-ui-bundle";
@import "../app/app.component";
```

Minimal:
```scss
/*
IMPORTANT: Stark styles are provided as SCSS styles so they should be imported in a SCSS file!
*/
@import "theme";
/* Stark styles */
@import "~@nationalbankbelgium/stark-ui/assets/styles/base";
@import "~@nationalbankbelgium/stark-ui/assets/theming/base-theme";
/* App */
@import "../app/app.component";
```

You can then import the files needed for your app.
```scss
/*
IMPORTANT: Stark styles are provided as SCSS styles so they should be imported in a SCSS file!
*/
@import "theme";
/* Stark styles */
@import "~@nationalbankbelgium/stark-ui/assets/styles/base";
@import "~@nationalbankbelgium/stark-ui/assets/theming/base-theme";
@import "~@nationalbankbelgium/stark-ui/src/modules/app-logo/components/app-logo.component";
@import "~@nationalbankbelgium/stark-ui/src/modules/app-logo/components/app-logo-theme";
/* App */
@import "../app/app.component";
```

Or use your own styling.
```scss
/*
IMPORTANT: Stark styles are provided as SCSS styles so they should be imported in a SCSS file!
*/
@import "theme";
/* Stark styles */
@import "~@nationalbankbelgium/stark-ui/assets/styles/base";
@import "~@nationalbankbelgium/stark-ui/assets/theming/base-theme";
@import "custom-stark-styles/buttons";
/* App */
@import "../app/app.component";
```

Reply:
I suggest restructuring it like this... what do you think?:

# Getting started

The recommended way to get started with **Stark** is to download the [starter](../../starter) and run `npm install`.

## Styling

The starter comes with most things preconfigured and all SCSS files needed for stark-ui are imported by default via `~@nationalbankbelgium/stark-ui/assets/stark-ui-bundle`:

```scss
/*
IMPORTANT: Stark styles are provided as SCSS styles so they should be imported in a SCSS file!
*/
@import "theme";
@import "~@nationalbankbelgium/stark-ui/assets/stark-ui-bundle";
/* App */
@import "../app/app.component";
```

You can also pick and choose which parts of the Stark styling/theming you want to use. Simply remove `@import "~@nationalbankbelgium/stark-ui/assets/stark-ui-bundle";` from `src/styles/styles.scss` and import only the desired files.

**IMPORTANT:** at least import `base.scss` and `base-theme.scss` since these provide you with a clean basis and some basics for styling/theming your app.

```scss
/*
IMPORTANT: Stark styles are provided as SCSS styles so they should be imported in a SCSS file!
*/
@import "theme";
/* Stark styles */
@import "~@nationalbankbelgium/stark-ui/assets/styles/base";
@import "~@nationalbankbelgium/stark-ui/assets/theming/base-theme";
/* Stark component styles */
@import "~@nationalbankbelgium/stark-ui/src/modules/app-logo/components/app-logo-theme";
@import "~@nationalbankbelgium/stark-ui/src/modules/app-logo/components/app-logo.component";
/* App */
@import "../app/app.component";
```

Or you can replace the Stark styling with your own:

```scss
/*
IMPORTANT: Stark styles are provided as SCSS styles so they should be imported in a SCSS file!
*/
@import "theme";
/* Stark styles */
@import "~@nationalbankbelgium/stark-ui/assets/styles/base";
@import "~@nationalbankbelgium/stark-ui/assets/theming/base-theme";
/* Stark component styles */
@import "~@nationalbankbelgium/stark-ui/src/modules/app-logo/components/app-logo-theme";
/* Custom Stark component styles */
@import "your-custom-stark-styles/app-logo.component";
/* App */
@import "../app/app.component";
```